src/AppKit/NSBezierPath.cs

```diff
@@ -99,6 +104,7 @@ public unsafe void AppendPathWithPoints (CGPoint[] points)
 				_AppendPathWithPoints ((IntPtr)ptr, points.Length);
 		}
 
+		[Obsolete ("Use 'AppendBezierPathWithCGGlyphs' instead.")]
```

Comment:
Might be better to point to the new Append (glyphs, font) method instead of AppendBezierPathWithCGGlyphs?